pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -337,15 +342,20 @@ lib.makeOverridable (
           ++ extraMakeFlags;
 
         installFlags =
-          [
-            "INSTALL_PATH=$(out)"
-          ]
-          ++ (optional isModular "INSTALL_MOD_PATH=$(out)")
+          [ "INSTALL_PATH=${placeholder "out"}" ]
           ++ optionals buildDTBs [
             "dtbs_install"
             "INSTALL_DTBS_PATH=$(out)/dtbs"
```

Comment:
This is now incorrect. (Could you split up the switch to `__structuredAttrs` into another commit?)

Reply:
How is it incorrect? Does `placeholder` not work with `__structuredAttrs`? Regardless, the change to `placeholder` here is on a piece of code technically not handled by the intended change of my PR, so I will change back to the `$(out)` style.